ambariclient/models.py

```diff
@@ -738,13 +760,109 @@ def restart(self, component_names=None):
             }))
         return self
 
+    def start(self, component_names=None):
+        """Starts this service."""
+        self.load(self.client.put(self.url, data={
+            "RequestInfo": {
+                "context": "_PARSE_.START.%s" % self.service_name,
+                "operation_level": {
+                    "level": "SERVICE",
+                    "cluster_name": self.cluster_name,
+                    "service_name": self.service_name,
+                }
+            },
+            "Body": {
+                "ServiceInfo": {"state": "STARTED"},
+            },
+        }))
+        return self
+
+    def stop(self, component_names=None):
+        """Stops this service."""
+        self.load(self.client.put(self.url, data={
+            "RequestInfo": {
+                "context": "_PARSE_.STOP.%s" % self.service_name,
+                "operation_level": {
+                    "level": "SERVICE",
+                    "cluster_name": self.cluster_name,
+                    "service_name": self.service_name,
+                }
+            },
+            "Body": {
+                "ServiceInfo": {"state": "INSTALLED"},
+            },
+        }))
+        return self
+
+
+class ConfigurationItemCollection(base.QueryableModelCollection):
+    @property
+    def url(self):
+        return self.parent.url
+
+    def create(self, tag=None, **kwargs):
```

Comment:
Can we add a little more documentation describing this function? I know Ambari has a nightmarish way of handling configuration changes, so sharing the knowledge you've clearly picked up the hard way would be useful to everyone :)

Reply:
gave few comments before and added a docstring with some more detials